packages/tasks/src/tasks/text2text-generation/inference.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
```

Comment:
Should we use this opportunity to unify `text-generation` and `text2text-generation`?